table_foosball/persistence.rb

```diff
@@ -0,0 +1,24 @@
+module Persistence
```

Comment:
I applaud your exploration of mixins here, but I think this abstraction is unhelpful.